refiner/scripts/seed_terminology_db.py

```diff
@@ -333,10 +337,40 @@ def populate_groupers_and_filters(self) -> None:
         self.logger.info("Finished populating groupers and filters tables")
 
 
+def dump_postgres_db_from_url(conn_str: str, output_file: str):
+    """
+    Dumps a PostgreSQL database using docker-compose and pg_dump.
+
+    Args:
+        conn_str: PostgreSQL connection string (e.g. postgresql://user:pass@host:port/dbname).
+        output_file: Path to save the dump file on the host.
+    """
+    # Grab user and database from connection string
+    match = re.match(r"postgresql://([^:]+):[^@]+@[^/]+/([^?]+)", conn_str)
+    if not match:
+        raise ValueError("Invalid connection string format.", conn_str)
+
+    user, dbname = match.groups()
+
+    # Compose the command
+    dump_cmd = f"docker-compose exec db pg_dump -U {user} {dbname} > {output_file}"
```

Comment:
i think that we've snuck in a dependency that isn't tracked; this is probably coming from the homebrew version of `docker-compse` (could you confirm with `which docker-compose`?)

meaning that we are relying on the python library `docker-compose` and not the go addition to the docker cli `docker compose`

it didn't work on my system because i don't have this installed in my env and the seeding failed. when i changed it i was able to get things working correctly.

```suggestion
dump_cmd = f"docker compose exec db pg_dump -U {user} {dbname} > {output_file}"
```

Reply:
Whoops 😬 Sorry about that! You're right that `compose` is supposed to now go through the `docker` binary. I have the old binary on my system that still works which is why I didn't catch this. 

Thank you!